### DISCUSSION TITLE
Custom Factions Combining Multiple Mods

### DISCUSSION THREAD

**Discussion by LazyTitan2** [Vdauphin/HeartsAndMinds#1478] [Question and answer] [answered]
Does anyone know of a way to add a new enemy faction "preset" if you will that combines sections from multiple existing mods?

IE the machine gunner from mod1 and marksmen from mod2.

Reasoning: 
Our server uses KP Liberation currently, wanting to switch to H&M but still want to be seeing the same opfor set we currently use, which combines multiple layers of 3cb, rhs, and S&S.

NOTE: There is a similar discussion to this related to CIV but im looking to do it for opfor on a much more granular scale

**Comment by Vdauphin** [comment, marked as answer]
Hello,
See http://vdauphin.github.io/HeartsAndMinds/Add-factions and this https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L600
Cheers
